From e465a1a43a4c7c3a0d6d153a90f814cadafe6307 Mon Sep 17 00:00:00 2001
From: Zahari Doychev <zahari.doychev@intel.com>
Date: Tue, 11 Dec 2018 17:08:20 +0100
Subject: [PATCH] include: tc_act: fix colmark action enum defines

Correct the tc colmark enum definitions to be more specific containing the
colmark in the name.

Signed-off-by: Zahari Doychev <zahari.doychev@intel.com>
---
 include/uapi/linux/tc_act/tc_colmark.h | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/include/uapi/linux/tc_act/tc_colmark.h b/include/uapi/linux/tc_act/tc_colmark.h
index 74f44da641fd..1b75f77577c4 100644
--- a/include/uapi/linux/tc_act/tc_colmark.h
+++ b/include/uapi/linux/tc_act/tc_colmark.h
@@ -11,19 +11,19 @@
 #define COLMARK_F_METER_TYPE		0x4
 
 enum tc_drop_precedence {
-	NO_MARKING,
-	INTERNAL = 1,
-	DEI = 2,
-	PCP_8P0D = 3,
-	PCP_7P1D = 4,
-	PCP_6P2D = 5,
-	PCP_5P3D = 6,
-	DSCP_AF = 7,
+	TC_COLMARK_NO_MARKING,
+	TC_COLMARK_INTERNAL = 1,
+	TC_COLMARK_DEI = 2,
+	TC_COLMARK_PCP_8P0D = 3,
+	TC_COLMARK_PCP_7P1D = 4,
+	TC_COLMARK_PCP_6P2D = 5,
+	TC_COLMARK_PCP_5P3D = 6,
+	TC_COLMARK_DSCP_AF = 7,
 };
 
 enum tc_meter_type {
-	srTCM,
-	trTCM,
+	TC_COLMARK_SRTCM,
+	TC_COLMARK_TRTCM,
 };
 
 struct tc_colmark {
